src/LoadableModule.cc

```diff
@@ -14,7 +14,8 @@
 // Note: We must use preprocessor instead of C ifs because if dlopen()
 // is seen by the static linker, the linker will complain.
 
-LoadableModule::LoadableModule(const String &aName): theName(aName), theHandle(nullptr)
+LoadableModule::LoadableModule(const SBuf &aName):
+        theName(aName)
```

Comment:
Just to fix [formatting](https://github.com/squid-cache/squid/actions/runs/8277010739/job/22646550395#step:4:153):

```suggestion
    theName(aName)
```

Reply:
done in [fd858aa](https://github.com/squid-cache/squid/pull/1738/commits/fd858aaa13035510319477ae023f7064982d4576)